content/manage-repos-pop.md

```diff
@@ -17,70 +24,62 @@ section: software
 tableOfContents: true
 ---
 
-## Accessing Repoman
+## Repositories and Sources
 
-Pop!_OS' tool for adding, removing and editing repositiories is called <u>Repoman</u> which is a part of the Pop!_Shop.
+Pop!\_OS comes pre-installed with a number of sources for installing additional software, but additional sources can be added as well, either by hand or by installing a third-party package. Sources for systemwide software packages are called _repositories_ (or "repos" for short), and sources for Flatpak software are called just _sources_.
 
-![Pop!_OS Shop](/images/manage-repos/pop-shop-button.png)
+## Repoman and System Repositories
 
-Click on the gear icon in the top right of the <u>Pop!_Shop</u> window. Once <u>Repoman</u> is opened, click on the **Extra Sources** tab there will be options to add repositories with the **Plus** button (on the bottom left). Select a repository from the list and then click on the **Edit** button (next to the **Plus** button) to edit the selected repository.
+Pop!\_OS' tool for adding, removing and editing repositiories and sources is called <u>Repoman</u>, and it is accessible through the Pop!_Shop.
```

Comment:
This would be "Pop!_OS's" unless "Pop!_OS" is a plural noun.

Reply:
Oops. Fixed.